plugins/modules/clickhouse_info.py

```diff
@@ -267,6 +288,97 @@ def get_roles(module, client):
     return roles_info
 
 
+def get_tables(module, client):
+    """Get tables.
+
+    Returns a dictionary with databases name as dictionary,
+    and the name of the table in this dictionary is the key.
+    """
+    query = ("SELECT database, name, uuid, engine, is_temporary, data_paths, "
+             "metadata_path, metadata_modification_time, dependencies_database, "
+             "dependencies_table, create_table_query, engine_full, partition_key, "
+             "sorting_key, primary_key, sampling_key, storage_policy, total_rows, total_bytes, "
+             "lifetime_rows, lifetime_bytes FROM system.tables")
+    result = execute_query(module, client, query)
+
+    if result == PRIV_ERR_CODE:
+        return {PRIV_ERR_CODE: "Not enough privileges"}
+
+    tables_info = {}
+    for row in result:
+        if row[0] not in tables_info:
+            tables_info[row[0]] = {}
+        tables_info[row[0]][row[1]] = {
+            "uuid": str(row[2]),
+            "engine": row[3],
+            "is_temporary": row[4],
+            "data_paths": row[5],
+            "metadata_path": row[6],
+            "metadata_modification_time": row[7],
+            "dependencies_database": row[8],
+            "dependencies_table": row[9],
+            "create_table_query": row[10],
+            "engine_full": row[11],
+            "partition_key": row[12],
+            "sorting_key": row[13],
+            "primary_key": row[14],
+            "sampling_key": row[15],
+            "storage_policy": row[16],
+            "total_rows": row[17],
+            "total_bytes": row[18],
+            "lifetime_rows": row[19],
+            "lifetime_bytes": row[20],
+        }
+
+    return tables_info
+
+
+def get_dictionaries(module, client):
+    """Get dictionaries.
+
+    Returns a dictionary with databases name as dictionary,
+    and the name of the 'dictionary' in this dictionary is the key.
```

Comment:
```suggestion
    Returns a dictionary with databases name as dictionary,
    and the name of the 'dictionary' in this dictionary is the key.
```
if my description change suggestions sound OK, should we also change it here?

Reply:
It may be better to leave this description for developers, it is more detailed and will not be included in the documentation. What do you think?

Reply:
Sure, works for me:) Maybe i don't even really understand what those dictionaries are about:)